spec/factories/courses.rb

```diff
@@ -210,7 +210,7 @@
       to_create { |instance| instance.save(validate: false) }
     end
 
-    trait :unpublished_with_primary_maths do
```

Comment:
1. Did you check there were no usages of this trait before renaming it?
2. Do we need to rename it? It would be a cleaner patch with the rename.

Reply:
This was one that I created in iteration 1